Example/Info.plist

```diff
@@ -64,6 +64,8 @@
 	<string>$(RMA_API_ENDPOINT)</string>
 	<key>RMAHostAppUserAgentInfo</key>
 	<string>$(RMA_USER_AGENT)</string>
+	<key>RMASSLKeyHash</key>
```

Comment:
Is it required to set this in the PLIST?

For our case, I don't think it makes sense to have a default value for the SSL pin. We want to dynamically set the pin each time that the `MiniAppConfig` is created. If no pin is provided in the `MiniAppConfig`, then we shouldn't use SSL pinning for that config. So I'm afraid having this set in the PLIST means that there is always a default and SSL pinning will always be used with all requests.

Reply:
@corycaywood no it's not required, and there and I paid attention to have no default value if not provided. This is mainly an help to us to provide CI values. If you want we cant undocument this.